ui/raidboss/data/07-dt/raid/r3n.ts

```diff
@@ -1,14 +1,175 @@
+import Outputs from '../../../../../resources/outputs';
+import { Responses } from '../../../../../resources/responses';
 import ZoneId from '../../../../../resources/zone_id';
 import { RaidbossData } from '../../../../../types/data';
 import { TriggerSet } from '../../../../../types/trigger';
 
 export type Data = RaidbossData;
 
+// MapEffect slots 01 - 08 are for the four towers.
+// MapEffect slot 09 is something to do with the referee, not important
+
+// Including headmarker data for savage reference
+const headMarkerData = {
+  // Vfx Path: com_share3t
+  stack: 'A1',
+  // Vfx Path: d1004turning_right_c0p
+  rotateClockwise: 'A7',
+  // Vfx Path: d1004turning_left_c0p
+  rotateCounterClockwise: 'A8',
+  // Vfx Path: m0676trg_tw_d0t1p
+  dualTankbuster: '103',
+} as const;
+console.assert(headMarkerData);
+
 const triggerSet: TriggerSet<Data> = {
   id: 'AacLightHeavyweightM3',
   zoneId: ZoneId.AacLightHeavyweightM3,
   timelineFile: 'r3n.txt',
-  triggers: [],
+  triggers: [
+    {
+      id: 'R3N Brutal Burn',
+      type: 'StartsUsing',
+      netRegex: { id: '9429', source: 'Brute Bomber', capture: true },
+      response: Responses.stackMarkerOn(),
+    },
+    {
+      id: 'R3N Brutal Impact',
+      type: 'StartsUsing',
+      netRegex: { id: '93D6', source: 'Brute Bomber', capture: false },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'R3N Knuckle Sandwich',
+      type: 'StartsUsing',
+      netRegex: { id: '93D5', source: 'Brute Bomber', capture: true },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'R3N Brutal Lariat 9AD4',
+      type: 'StartsUsing',
+      netRegex: { id: '9AD4', source: 'Brute Bomber', capture: false },
+      response: Responses.goEast(),
+    },
+    {
+      id: 'R3N Brutal Lariat 9AD5',
+      type: 'StartsUsing',
+      netRegex: { id: '9AD5', source: 'Brute Bomber', capture: false },
+      response: Responses.goWest(),
+    },
+    {
+      id: 'R3N Murderous Mist',
+      type: 'StartsUsing',
+      netRegex: { id: '93B5', source: 'Brute Bomber', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'R3N Barbarous Barrage',
+      type: 'StartsUsing',
+      netRegex: { id: '93B2', source: 'Brute Bomber', capture: false },
+      infoText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Knockback Towers',
```

Comment:
I didn't think to test it when I was in there, this knockback can be nullified, right? It has the correct indicator for it.

Reply:
Yes, it can.